.travis.yml

```diff
@@ -34,8 +34,8 @@ jdk:
   - openjdk8
 
 env:
-  - FLINK_VERSION="1.10.0" SCALA_VERSION="2.11"
-  - FLINK_VERSION="1.10.0" SCALA_VERSION="2.12"
+  - FLINK_VERSION="1.11.1" SCALA_VERSION="2.11"
```

Comment:
could you change to same version of pom (1.11.2)

Reply:
done